### PR TITLE
add reference node

### PR DESCRIPTION
**Pull request by alavieille**
[OO-FEATURE] Add use reference when node is used in node

https://github.com/open-orchestra/open-orchestra-display-bundle/pull/247
https://github.com/open-orchestra/open-orchestra-model-interface/pull/243
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/60
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/667
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1934
https://github.com/open-orchestra/open-orchestra/pull/1053

### CODE REVIEW

File: Backoffice/Reference/Strategies/NodeInNodeReferenceStrategy.php [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    }
+
+    /**
+     * @param $str
+     *
+     * @return boolean
+     */
+    protected function hasInternalLinkBBCode($str)
+    {
+        $internalLinkBBCode = '/\[' . InternalLinkDefinition::TAG_NAME . '(\=\{.*\})?].*\[\/'
```
Comment (itkg-nanne) [inline Backoffice/Reference/Strategies/NodeInNodeReferenceStrategy.php]: I would use "'/\[' . InternalLinkDefinition::TAG_NAME . '[^]]*].*?\[\/'. InternalLinkDefinition::TAG_NAME . '\]/m'"

**Comment by alavieille** [comment]
@itkg-nanne pr updated

**Comment by itkg-nanne** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Pull request closed without merging**